packages/wrangler/src/deploy/deploy.ts

```diff
@@ -1142,6 +1146,15 @@ function updateQueueConsumers(config: Config): Promise<string[]>[] {
 	});
 }
 
+// TODO(soon): workerd requires python modules to be named without a file extension
+// We should remove this restriction
```

Comment:
Already fixed in https://github.com/cloudflare/workerd/pull/1597, we'll just need to make a new release and update to it.